routes/plant/garden.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const router = express.Router();
+
+// Import schemas
+const User = require('../../schemas/user');
+const Plant = require('../../schemas/plant');
+
+// Authenticate user
+const authenticateUser = async (req, res, next) => {
+    const accessToken = req.header("Authorization");
+    try {
+      const user = await User.findOne({accessToken: accessToken});
+      if (user) {
+        next();
+      } else {
+        res.status(401).json({
+          success: false,
+          response: "Please log in"
+        })
+      } 
+    } catch (e) {
+      res.status(500).json({
+        success: false,
+        response: e
+      })
+    }
+  };
+
+router.get("/users/:username/garden", authenticateUser);
+router.get("/users/:username/garden", async (req, res) => {
```

Comment:
In deleteplants & editpants the route was 
":username/garden/:plantId"

while here in garden is "/users/:username/garden"